cellprofiler/modules/tram.py

```diff
@@ -0,0 +1,673 @@
+from cellprofiler.modules import trackobjects
+import numpy as np
+import itertools as it
+import cellprofiler.module as cpm
+import cellprofiler.setting as cps
+import cellprofiler.measurement as cpmeas
```

Comment:
Please refer to our [style guide](https://github.com/CellProfiler/CellProfiler/blob/master/CONTRIBUTING.md#style-guide) for style guidelines.